config/src/config/test_data/validator.yaml

```diff
@@ -49,3 +49,9 @@ validator_network:
                 from_disk: "/full/path/to/token"
     mutual_authentication: true
     max_frame_size: 8388608 # 8 MiB
+    network_address_key_backend:
```

Comment:
there's 4 identical backend in this file, should we consolidate them in a single backend config per validator?

Reply:
There's a note somewhere to explore that... the problem is the routing of that information since we would ideally like the NetworkConfig struct to be independent of other Config structs.